editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -268,6 +273,69 @@ export var ${BakedInStoryboardVariableName} = (props) => {
 `
 }
 
+async function doDblClickTest(editor: EditorRenderResult, testId: string): Promise<HTMLElement> {
+  const canvasControlsLayer = editor.renderedDOM.getByTestId(CanvasControlsContainerID)
+  const div = editor.renderedDOM.getByTestId('mydiv')
+  const divBounds = div.getBoundingClientRect()
+  const divCorner = {
+    x: divBounds.x + 50,
+    y: divBounds.y + 40,
+  }
+
+  mouseClickAtPoint(canvasControlsLayer, divCorner)
+
+  const nineBlockControlSegment = editor.renderedDOM.getByTestId(testId)
+
+  mouseClickAtPoint(nineBlockControlSegment, { x: 2, y: 30 }, { eventOptions: { detail: 2 } })
```

Comment:
We have a `mouseDoubleClickAtPoint` which would be better to use here as it would fire all of the events